README.md

```diff
@@ -17,6 +21,7 @@ The following will take the yaml file and produce a scanapi-report.html file as
 - name: Run automated API tests
   uses: scanapi/github-action@v1
   with:
+    scanapi_version: '==2.0.0'
```

Comment:
Is there a reason for removing this feature?

Reply:
> Everything looks good to me. I would probably add a feature to specify the version in PIP version specifier syntax ('==1.2.3', '~=1.2', etc.) as it was added by @ShubhamSood1406. This would add the capability for users to automatically use the latest version if the patch version is changed and not to update when major updates (with braking changes) occur.

Oh, that makes sense, now I got the point of the way you implemented it on poetry-publish. I agree with you 👍